examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java

```diff
@@ -1,7 +1,40 @@
 package dev.selenium.interactions;
 
-import dev.selenium.BaseTest;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class NavigationTest extends BaseTest {
+public class browserNavigation {
```

Comment:
`public class NavigationTest {`

Reply:
Yes thats obvious. Not sure why this code is showing . I had changed . Will check and update in sometime . Again.

Reply:
corrected the mistake. please check. sorry about this.